### DISCUSSION TITLE
Edit project of an activity

### DISCUSSION THREAD

**Discussion by RokkuCode** [kimai/kimai#3138] [Questions & Answers] [answered]
Hi,

On creating a new activity it is possible to select a project. But on editing an activity its not - or am I blind?

Kind regards,
Rokku

**Comment by kevinpapst** [comment, marked as answer]
It is not possible, correct. 
Way too many problems that can arise from changing the project, eg for rates and existing timesheets.

**Reply by RokkuCode** [reply]
Ok. Thanks for your fast response :)
